Makefile

```diff
@@ -14,6 +16,13 @@ lint: get_deps
 test: get_deps
 	yarn test
 
+test_integration:
+	- ${DOCKER_COMPOSE_TEST} down -v
```

Comment:
If we're stopping containers on L24 after this commands been run do we need to account for them being up on start?

Reply:
i think this is more for local development. sometimes a test fails or you are in an inconsistent state with the containers so this makes sure it starts fresh.